src/machine/machine_rp2040_usb_fix_usb_device_enumeration.go

```diff
@@ -0,0 +1,112 @@
+//go:build rp2040
```

Comment:
@sago35 please remove the older format build tag here. See #3348

Reply:
Old buildtag removed.